### PR TITLE
Always pull latest Sauce Connect version if not specified otherwise

### LINKED ISSUE #93: Pull always latest Sauce Connect version if not specified otherwise
We should update the `sc` command to always pull the latest Sauce Connect version from https://saucelabs.com/versions.json.

### PR DESCRIPTION
**Pull request by martinfrancois**
Fallback to hardcoded version in case the API call fails.

Closes #93

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -291,6 +294,20 @@ export default class SauceLabs {
         })
     }
 
+    async _getLatestSauceConnectVersion () {
+        const uri = SAUCE_CONNECT_VERSIONS_ENDPOINT
+        const responseType = 'json'
+
```
Comment (christian-bromann) [inline src/index.js:300]: No need to define them into const as we don't modify them here.

**Review by christian-bromann — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (martinfrancois) [reply src/index.js]: True, just inlined them!

**Review by christian-bromann — APPROVED** [review]
👍  thank you so much!

**Comment by martinfrancois** [comment]
You're welcome! 😃

**Pull request merged**